draft-ietf-quic-http.md

```diff
@@ -472,145 +475,142 @@ or complete response but does not abort reading, clients SHOULD continue sending
 the body of the request and close the stream normally.
 
 
-### Header Formatting and Compression {#header-formatting}
+### Field Formatting and Compression {#header-formatting}
 
-HTTP message headers carry information as a series of key-value pairs, called
-header fields. For a listing of registered HTTP header fields, see the "Message
-Header Field" registry maintained at
-<https://www.iana.org/assignments/message-headers>.
+HTTP messages carry metadata as a series of key-value pairs, called HTTP fields.
+For a listing of registered HTTP fields, see the "Hypertext Transfer Protocol
+(HTTP) Field Name Registry" maintained at
+<https://www.iana.org/assignments/http-fields/>.
 
-Just as in previous versions of HTTP, header field names are strings of ASCII
+Just as in previous versions of HTTP, field names are strings of ASCII
```

Comment:
Hm. It's not _all_ ASCII characters.

Reply:
Field names are defined to be `token`, which is `1*tchar`, which is "any visible US-ASCII character" "except delimiters."  Are there non-ASCII values in that?

Reply:
_except delimiters_. This spec should either just refer to the authoritative definition (preferred), or accurately describe them.

Reply:
Ah, I misunderstood the direction of your comment. I'll update in the morning.

Reply:
Looks good.